### ISSUE TITLE
How many PD / PDFS are there in the knee validation set?

### ISSUE THREAD

**Issue by tianweiy** [facebookresearch/fastMRI#46]
Hi, I wonder what is the ratio between PD / PDFS images in the validation set. I want to use this to convert the validation accuracy(now PD / PDFS individually) to a single value
